libs/langgraph/langgraph/prebuilt/chat_agent_executor.py

```diff
@@ -229,13 +235,13 @@ def _validate_chat_history(
 
 
 @deprecated_parameter("messages_modifier", "0.1.9", "state_modifier", removal="0.3.0")
```

Comment:
lets remove this one, no?

Reply:
if it's not getting in the way for users (not affecting adding confusing things to API reference or the doc string seeing in an interactive env), we should consider removing the deprecation warning.

okay for the internal code to be backwards compatible -- we just care that things that are user facing aren't confusing

Reply:
sure, we can, but we also should remove it in 0.3 and simplify the backwards compatibility code as a result